app/models/package_manager/maven.rb

```diff
@@ -134,5 +138,34 @@ def self.project_name(name)
         artifactId: sections[1]
       }
     end
+
+    def self.get_pom(group_id, artifact_id, version, seen=[])
+      seen << [group_id, artifact_id, version]
+
+      xml = get_xml(
+        "http://repo1.maven.org/maven2/#{group_id}/#{artifact_id}/#{version}/#{artifact_id}-#{version}.pom"
+      )
+
+      next_group_id = xml.locate('distributionManagement/relocation/groupId').first&.nodes&.first || group_id
```

Comment:
Spicy

Reply:
I'd actually like to simplify this, if possible - thoughts?